client/src/pages/admin/pages/plugins.tsx

```diff
@@ -244,33 +196,30 @@ export const PluginAdd = () => {
             />
           </div>
           <div className="modal-action">
-            <button
-              className="btn btn-primary"
-              onClick={(e) => {
-                e.preventDefault();
-                setOpen(false);
-              }}
-            >
-              Close
-            </button>
             <button className="btn btn-primary" type="submit" aria-label="create plugin">
               Save
             </button>
           </div>
         </form>
-      </Modal>
     </>
   );
 };
 
 export const PluginRow = ({ plugin, removePlugin }: PluginRowProps) => {
+  const [showModal, setShowModal] = useState(false);
```

Comment:
Is this being used for both dialogs? Will that not make both dialogs show when clicking either 'edit' or 'add' button?

Reply:
Nvm I can see these are separate methods, I would change the name so you associate it to the edit button, so it's showEditModal, setShowEditModal for example